bitrise.yml

```diff
@@ -52,6 +52,9 @@ workflows:
             echo "-- build-for-testing --"
             mkdir DerivedData
 
+            brew install xcodes
+            xcodes runtimes install "iOS 18.2"
```

Comment:
As commented on Slack, adding all these installations may increase the build + test time. If there is not a rush to move to xcode 16.3 let's wait until we can use the simulators that come by default so that we avoid all these workarounds.